src/snapred/backend/dao/ingredients/DiffractionCalibrationIngredients.py

```diff
@@ -20,4 +21,4 @@ class DiffractionCalibrationIngredients(BaseModel):
     groupedPeakLists: List[GroupPeakList]
     calPath: str
     convergenceThreshold: float
-    maxOffset: float = 10.0
+    maxOffset: float = Config["constants.DiffractionCalibrationMaxOffset"]
```

Comment:
I think this one is already listed in `calibration.diffraction.maximumOffsets`, but it may have been added after you started this work

Reply:
Thanks, this has been updated.